lib/block.ml

```diff
@@ -125,6 +129,14 @@ let pp_lines syntax =
 
 let pp_contents ?syntax ppf t = Fmt.pf ppf "%a\n" (pp_lines syntax) t.contents
 
+let pp_errors ppf t =
+  match t.value with
+  | OCaml { errors; _ } when List.length errors > 0 ->
+      Fmt.string ppf "```\n";
```

Comment:
Is there any particular reason why `pp_errors` should be closing the above block?

Reply:
No particular reason, just that I called `pp_errors` before `pp_footer` but that's not very pretty indeed, will fix that (and squash)